src/NBitcoin/Networks.cs

```diff
@@ -137,192 +280,219 @@ private static Network InitStratisMain()
 
             consensus.LastPOWBlock = 12500;
 
-			consensus.ProofOfStakeLimit =   new BigInteger(uint256.Parse("00000fffffffffffffffffffffffffffffffffffffffffffffffffffffffffff").ToBytes(false));
-			consensus.ProofOfStakeLimitV2 = new BigInteger(uint256.Parse("000000000000ffffffffffffffffffffffffffffffffffffffffffffffffffff").ToBytes(false));
+            consensus.ProofOfStakeLimit =   new BigInteger(uint256.Parse("00000fffffffffffffffffffffffffffffffffffffffffffffffffffffffffff").ToBytes(false));
+            consensus.ProofOfStakeLimitV2 = new BigInteger(uint256.Parse("000000000000ffffffffffffffffffffffffffffffffffffffffffffffffffff").ToBytes(false));
 
-			consensus.CoinType = 105;
+            consensus.CoinType = 105;
 
-			var genesis = CreateStratisGenesisBlock(1470467000, 1831645, 0x1e0fffff, 1, Money.Zero);
-			consensus.HashGenesisBlock = genesis.GetHash();
+            Block genesis = CreateStratisGenesisBlock(1470467000, 1831645, 0x1e0fffff, 1, Money.Zero);
+            consensus.HashGenesisBlock = genesis.GetHash();
 
-			// The message start string is designed to be unlikely to occur in normal data.
-			// The characters are rarely used upper ASCII, not valid as UTF-8, and produce
-			// a large 4-byte int at any alignment.
-			var pchMessageStart = new byte[4];
+            // The message start string is designed to be unlikely to occur in normal data.
+            // The characters are rarely used upper ASCII, not valid as UTF-8, and produce
+            // a large 4-byte int at any alignment.
+            var pchMessageStart = new byte[4];
```

Comment:
when possible, get rid of this pch prefix (or other hungarian whatever prefixes)

Reply:
ah yea done